README.md

```diff
@@ -73,8 +73,4 @@ This performs several optimizations which can significantly reduce the final siz
 
 ## License
 
-Copyright 2020 Anchor Protocol
-
-Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0. Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-
-See the License for the specific language governing permissions and limitations under the License.
+See (License)[./LICENSE]
```

Comment:
I think this should be`[License](./LICENSE)`. I will fix that and push it to the branch.